models/task.go

```diff
@@ -1,16 +1,15 @@
 package models
 
 import (
-	"time"
-
 	"github.com/fatih/structs"
+	"time"
 )
 
 // Task represents a task in the task scheduler.
 type Task struct {
-	ID          string    `json:"id"`
-	Name        string    `json:"name"`
-	CreatedAt   time.Time `json:"created_at"`
+	ID          int       `json:"id"`
+	Name        string    `json:"name1"`
+	Created     time.Time `json:"created"`
```

Comment:
mysqldb_test.go:66: SaveTask() error = failed to save a task: could not find name created_at in
it was giving this error so i removed "_at"

Reply:
All confusion from caused here 	query := `
		INSERT INTO tasks ( name, createdAt, status, description, deadline, priority, assignee)
		VALUES (:name, :createdat, :status, :description, :deadline, :priority, :assignee)
	`if i change createdat in values to created_at or createdAt it gives error i don't understand why it is like that

Reply:
In that case, did you check if the column `created_at` existed in the database or not?
It would be helpful to review the database schema and ensure that the field names in the code match the corresponding column names in the database. Consistency between the code and the database will help avoid potential errors and make the code more intuitive.

Reply:
yes i had checked gonna recheck

Reply:
i had this column createdAt in database  query := INSERT INTO tasks ( name, createdAt, status, description, deadline, priority, assignee) as you can see here the roblem is caused by place holder values

Reply:
make sure you are adding relevant tags in your Models for JSON/BSON representation that correspond to your database columns or fields.